resources/device_types/distech_ecy-s1000/module_config.json

```diff
@@ -91,7 +85,7 @@
         },
         "161": {
```

Comment:
Some is off here because I don't think the module_config for a device_type should be explicitly specifying some of these values.  E.g., -- it should assume there's no server ports and then only explicitly say which ones are allowed. Same with the test configurations.. I forget what the "discover" test is, but why is it explicitly enabled for *this* device type?

Reply:
The discover module has at the moment a test to extract the firmware version from a bacnet scan. It is intended to grow with the addition of more tests to discover features from the devices, but haven't managed to find the time to add more yet.

About the module_config.json files, there is one class for devices (intended to go inside the mac_address folder for each device) and another one for the system (intended to go inside local/site).
I've added the TCP and UDP ports in my record that are associated with IoT and building control devices. If we don't add them somewhere, the nmap test is restricted to the few previously specified and becomes very narrow in scope. Perhaps a better way to add these ports to be checked is the module_config.json for the system and then the device ones has only the ports that matter to be open for that device?

The other thing I've provided is a minimal device module_config.json that just expects the user to add the device type and the device specific information.

This is all I have to say, but can start another document if this discussion becomes deeper than this, sure.

Reply:
I'm wondering if we need another section which is a device class? Sounds like there some information that needs to be for most devices of that class -- and either it gets replicated for each type, which is bad, or globalized -- which is also bad.

I'll leave it up to you if you think you want to submit this one as is and then we iterate, or fix it all up now!

Reply:
Thank you @grafnu I'd prefer to submit as it is and then iterate if OK.